include/onnxruntime/core/common/profiler_common.h

```diff
@@ -27,19 +27,74 @@ static constexpr const char* event_categor_names_[EVENT_CATEGORY_MAX] = {
 
 // Timing record for all events.
 struct EventRecord {
+  EventRecord() = default;
```

Comment:
>EventRecord() [](http://example.com/codeflow?start=2&length=13)

when enabling default constructor, members are better to have corresponding initializers along with declaration.

Reply:
Done!